test/fixtures/mysql-ha/versions.tf

```diff
@@ -15,5 +15,9 @@
  */
 
 terraform {
-  required_version = ">= 0.12"
+  required_version = "~> 0.12.0"
+  required_providers {
```

Comment:
Required providers should only be configured on functional modules and submodules, not examples or fixtures.

Reply:
Moved to submodules